experimental/generation/generator/packages/library/src/substitutions.ts

```diff
@@ -111,20 +123,19 @@ function substitutions(path: string, bindings: any, copies?: number, seed?: stri
             if (line.trim() === '') {
                 result.push(line)
             } else {
-                let generated = new Set<string>()
-                let all = line.match(/\${[^}*]+\*}/g) || []
-                for (let [key, variable] of state) {
+                const all = line.match(/\${[^}*]+\*}/g) || []
```

Comment:
`??` is generally preferred to `||`, though that's more of a nitpick in this case.